oxidation/libparsec/crates/protocol/schema/authenticated_cmds/block_create.json

```diff
@@ -1,46 +1,53 @@
-{
-    "label": "BlockCreate",
-    "req": {
-        "cmd": "block_create",
-        "other_fields": [
+[
```

Comment:
Why add an extra array ?

Reply:
That's explain in #2834, if you have a question related to that issue ask the question

Reply:
Holy :scream: